samples/sh/hello-world/set.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/bash
+printf "$contentMatch" >> "$path"
```

Comment:
Missing trailing newline.

```suggestion
printf "$contentMatch" >> "$path"

```